deploy/kubernetes/overlays/stable/kustomization.yaml

```diff
@@ -4,7 +4,7 @@ bases:
   - ../../base
 images:
   - name: gcr.io/k8s-staging-cloud-provider-ibm/ibm-powervs-block-csi-driver
-    newTag: main
+    newTag: test-image-v1
```

Comment:
lets add next alpha version to be realistic,

Reply:
fixed